packages/react-composites/src/composites/MeetingComposite/NotificationIcon.tsx

```diff
@@ -21,7 +21,7 @@ export const NotificationIcon = (props: NotificationIconProps): JSX.Element => {
       <Icon styles={NotificationIconStyles} iconName="NotificationIcon" />
       <Stack styles={NotificationTextStyles}>
         {chatMessagesCount > 0 && chatMessagesCount < 9 ? <>{chatMessagesCount}</> : <></>}
-        {chatMessagesCount > 9 ? <>+9</> : <></>}
+        {chatMessagesCount >= 9 ? <>9+</> : <></>}
```

Comment:
Heh ;)

Should we be localizing numbers as well (so this would come from the strings prop).

Reply:
^^ Not for this PR.